client/src/pages/Campaign.js

```diff
@@ -137,22 +139,22 @@ const Campaign = (props) => {
     });
   }
 
-  const handleEmailTemplates = data => {
-    const emailTemplatesData = data['templates'];
-    const emailTemplates = [];
-    for(let emailTemplate of emailTemplatesData) {
-      emailTemplates.push({
-        id : emailTemplate.id,
-        name : emailTemplate.name,
-        type : emailTemplate.type,
-        subject : emailTemplate.subject,
-        body : emailTemplate.body
+  const handleTemplates = data => {
+    const templatesData = data['templates'];
+    const templates = [];
+    for(let template of templatesData) {
+      templates.push({
+        id : template.id,
+        name : template.name,
+        type : template.type,
+        subject : template.subject,
+        body : template.body
       })
     }
-    setEmailTemplates(emailTemplates);
+    setTemplates(templates);
   }
 
-  const getEmailTemplates = async () => {
+  const getTemplates = async () => {
     await fetch('/templates', {

```

Comment:
Maybe use the apiRequest from utils?

Reply:
Ah yea, I should transition into that soon